lib/config.go

```diff
@@ -27,6 +27,9 @@ const (
 	defaultTransport    = "docker://"
 	apparmorProfileName = "crio-default"
 	cgroupManager       = oci.CgroupfsCgroupsManager
+	runcRuntime         = "runc"
```

Comment:
Can  you call these 
        defaultRuntime         = "runc"
	defaultRuntimePath     = "/usr/bin/runc"
	defaultRuntimeType     = "oci"

Reply:
Done